ci-operator/config/openshift/loki/openshift-loki-upstream-v2.6.1.yaml

```diff
@@ -0,0 +1,65 @@
+base_images:
+  base:
+    name: "4.11"
+    namespace: ocp
+    tag: base
+  lokistack-gateway:
+    name: 5.y
+    namespace: logging
+    tag: lokistack-gateway
```

Comment:
Does this build need `lokistack-gateway` and `opa-openshift`? I don't think we use it in the tests as upstream Loki does not know anything about these components.